python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -75,23 +117,23 @@ def __iter__(self):
 
     def residuals(self, fiberId=None):
         """Return wavelength residuals (nm)
-
         Parameters
         ----------
         fiberId : `int`, optional
             Fiber identifier to select.
-
         Returns
         -------
         residuals : `numpy.ndarray` of `float`
             Wavelength residuals (nm): measured - actual.
         """
-        return np.array([ll.measuredWavelength - ll.actualWavelength for ll in self.lines if
+
+        return np.array([ll.fitWavelength - ll.reflines for ll in self.lines if
                          fiberId is None or ll.fiberId == fiberId])
 
+
     def mean(self, fiberId=None):
-        """Return the mean of wavelength residuals (nm)
 
+        """Return the mean of wavelength residuals (nm)
```

Comment:
Remove blank line between `def` and docstring, and reinsert blank line between one-line summary and Parameters.